src/index.html

```diff
@@ -5,8 +5,356 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&family=Roboto:wght@400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="navigation">
+      <img
+        src="/images/logo.png"
+        alt="icon"
+        class="navigation__image"
+      >
+      <nav class="nav-container">
+        <ul class="nav-container__list">
+          <li class="nav-container__item">
+            <a
+              class="nav-container__link nav-container__link--active"
+              href="#"
```

Comment:
```suggestion
              href="#apple"
```

update links name everywhere